### DISCUSSION TITLE
How to import module with absolute import dynamically

### DISCUSSION THREAD

**Discussion by anoop-gupt** [vercel/next.js#29452] [Help]
Hi,
I would like to use Dynamic Import with absolute import for example
`const DynamicComponent = dynamic(() => import {MyComponent} from @Libs/Core );`

But things are not working, any pointer would be really helpful.

**Comment by wwwdepot** [comment]
It should be 

```js
const DynamicComponent = dynamic(() => import("@Libs/Core").then(mod => mod.MyComponent));
```
[See Docs - With Named Exports](https://nextjs.org/docs/advanced-features/dynamic-import#with-named-exports)
